buildkite/scripts/build_tarball.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+GOOS=$1 GOARCH=$2 make build-bacalhau-tgz
```

Comment:
I looked at the build logs https://buildkite.com/expanso/bacalhau-golang/builds/291#01910786-63c9-4a7e-b38d-d12c63b50496

some observations:
- It seems we are building the webui everytime. This is an issue with make target setup
- I see some failures at the end related to signing the builds because of missing /tmp/private.pem. Check https://github.com/bacalhau-project/bacalhau/blob/e71b30bf47fc6bfc9e19d4be976f9996e3426d88/.circleci/config.yml#L317 on how we were doing things with circleci

Reply:
Yess ! I want to have at least one green build with buildkite before I merge this. I am debugging those.